components/Episode/EpisodeCard.js

```diff
@@ -61,18 +66,25 @@ export default function EpisodeCard({
       ) : (
         <NoContentMessage>keine Beschreibung vorhanden</NoContentMessage>
       )}
+      {speakers?.length > 0 ? (
+        <Speakers speakers={speakers} />
+      ) : (
+        <NoContentMessage>keine Stimmen angegeben</NoContentMessage>
+      )}
       {chapters?.length > 0 ? (
         <Chapters chapters={chapters} />
       ) : (
         <NoContentMessage>
-          {parts ? "" : "keine Kapitelliste vorhanden"}
+          {parts?.length > 0 ? "" : "keine Kapitelliste vorhanden"}
         </NoContentMessage>
```

Comment:
Why the content message from the chapters depends on the parts? What if I have no chapters but parts? Then nothing will be displayed. Maybe I just understand the difference between parts and chapters.